rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -183,7 +181,7 @@ error_code sys_rsx_context_iomap(u32 context_id, u32 io, u32 ea, u32 size, u64 f
 		return CELL_EINVAL;
 	}
 
-	vm::reader_lock rlock;
+	vm::writer_lock lock(0);
```

Comment:
Am I missing something? Why do we need to hold a full vm lock?

Reply:
Adding a vm reader for rsx_iounmap instead also suffices.

Reply:
Much better.